ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/console/page/content/clientlibs/js/paths.js

```diff
@@ -14,8 +14,40 @@
 $(function () {
     "use strict";
 
+    $(document).on("click", '.js-create-package-activator', function(event) {
+        var dialog = document.querySelector('#createPackageDialog');
+        if (!dialog) {
+            var path = $('.js-create-package-activator').data('path');
+            $.ajax({
+                type: "GET",
+                url: '/apps/etoolbox-backpack/admin/console/page/content/createpagedialog.html?paths=' + path,
+                success: function(data) {
+                    var doc = new DOMParser().parseFromString(data, 'text/html');
+                    document.body.appendChild(doc.body.firstChild);
+                }
+            });
+        }
+        setTimeout(function() {
+            dialog = document.querySelector('#createPackageDialog');
+            fillPaths();
+            dialog.show();
+        }, 300);
```

Comment:
BTW, what this timeout is for? It used to be 100 yesterday, now it's 300.

I noticed that you query for the same object twice: first in line 18, and second, in line 31. 
I may be mistaken but it looks like you are not sure that some node is already there so you just sit and wait for it for an arbitrary period of time (which may vary depending on the end-user's machine). 

You may want to use instead [Coral.commons.ready](https://developer.adobe.com/experience-manager/reference-materials/6-5/coral-ui/coralui3/Coral.commons.html#Coral.commons:.ready) (if waiting for a Coral component)

or [dialog-ready](https://stackoverflow.com/questions/49779675/dialog-ready-event-does-not-fire-when-dialog-opens-in-aem-6-1)

or [foundation-contentloaded](https://developer.adobe.com/experience-manager/reference-materials/6-5/granite-ui/api/jcr_root/libs/granite/ui/components/foundation/clientlibs/foundation/vocabulary/contentloaded.html)

Timer is a last resort.